plantcv/plantcv/cluster_contour_splitimg.py

```diff
@@ -119,9 +119,9 @@ def cluster_contour_splitimg(rgb_img, grouped_contour_indexes, contours, hierarc
         else:
             savename = os.path.join(".", group_names[y])
             savename1 = os.path.join(".", group_names1[y])
-        iy, ix, iz = np.shape(rgb_img)
+        iy, ix, iz = np.shape(img)
```

Comment:
```suggestion
        iy, ix = np.shape(img)[:2]
```

It looks like we never use the variable `iz` so we can just use a slice to return only the first two dimensions, then it won't matter if `img` is RGB or grayscale.